File: javalin-gradle-java-basic/src/test/java/org/example/service/MyServiceTest.java
package org.example.service;

import io.dinject.SystemContext;
import org.junit.Test;

import static org.junit.Assert.*;

public class MyServiceTest {

  @Test
  public void hi_using_di() {

    final MyService bean = SystemContext.getBean(MyService.class);

    assertEquals("hello", bean.hi());
  }
}

File: javalin-gradle-kotlin-basic/src/test/kotlin/org/example/service/MyServiceTest.kt
package org.example.service

import io.dinject.SystemContext
import org.junit.Test

import org.junit.Assert.*

class MyServiceTest {

    @Test
    fun hello() {

      val bean = SystemContext.getBean(MyService::class.java)
      assertEquals("hello", bean.hello())
    }
}
